### DISCUSSION TITLE
How to display price with discount in BO produt list

### DISCUSSION THREAD

**Discussion by sirius82** [PrestaShop/PrestaShop#33206] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hello,
With PS 1.6, final prices with discount were displayed in B.O product list. Not with PS 1.7.8. Final price is only price with tax included.
I founded any tutos but not for this PS version.

### Expected behavior

I wish to add a column in BO product list with final price with discount like with PS 1.6. it's easier to find wich products have a reduction.

### Steps to reproduce

1. Go to a product
2. Add a specific price (with % or amount reduction)
3. Go to product list in BO
4. Specific price is displayed in a new column. It's possible to range by specific price or to filter only by products with specific price.

### PrestaShop version(s) where the bug happened

1.7.8.6

### PHP version(s) where the bug happened

7.4

**Comment by jolelievre** [comment]
Hi @sirius82,

the problem is that quite often the discount depend on the customer context: its customer group, country, currency, ... All this context is completely different in the BO since you are connected as an Employee with different settings, so I'm afraid this would lead to invalid or unexpected prices in most cases.

So I'm not sure this should be in the core by default, however you could create a module that overrides the product grid to add a custom column relying on the hooks from the grid system.

Another complexity is that specific price are computed by PHP code they are not directly accessible in the DB so order and paginating on such prices can be quite challenging.

**Comment by kpodemski** [comment]
I transferred this issue to Discussions since it's a feature request/question, not a bug report.

**Comment by sirius82** [comment]
hello @jolelievre 
I'm sorry, I was sure to have reply but I don't see my reply.

Is the context different between PS 1.6 and P.S 1.7 ? Because with I.6, I had this column with final price (price - reduction = final price)
Why not with PS 1.7 ?
If it's so complicated, I would have a Yes or No column on whether there's a specific price or not. But I would like to know how.

Thanks a lot !

**Comment by sirius82** [comment]
Hello,
Is it possible to have some help to add a column with reduced price in BO ?

Thanks
